include/swift/Remote/MetadataReader.h

```diff
@@ -264,6 +264,14 @@ class MetadataReader {
     return start;
   }
 
+  /// Given a pointer to an address, attemp to read the pointed value.
+  llvm::Optional<StoredPointer> readPointedValue(StoredPointer Address) {
```

Comment:
`readPointerValue`, maybe?  You use `pointed` throughout this patch, but I don't think we really use that term anywhere else.

Reply:
Sure, I can change that :)